src/rlx_tar.erl

```diff
@@ -181,9 +197,18 @@ filter(_) ->
 
 %% if `include_src' is true then include the `src' and `include' directories of each application
 app_dirs(State) ->
-    case rlx_state:get(State, include_src, true) of
+    case include_src_or_default(State) of
         false ->
             [];
         true ->
             [include, src]
     end.
+
+%% when running `tar' the default is to exclude src
```

Comment:
I wanted to enable systools `src_tests` by default. It checks that the beams are not older than the erl files. To do this check the src has to be in `rel/<relname>/lib` but doesn't need to go into the tarball.

So in my rebar3_test example app it looks like this if I edit a module in a dependency (in this case elli) since rebar3 does not compile manual changes in a dependency's source:

```
../rebar3/rebar3 release                                                                                   
===> Verifying dependencies...
===> Compiling relx_overlays
===> Warnings generating release:
*WARNING* Missing application sasl. Can not upgrade with this release
*WARNING* elli: Object code (elli) out of date
```

Reply:
Since we do always compile the project app before release and discourage changing source in deps because we don't recompile it might not be useful and so I'm still unsure if it matters and may not bother with this.